armada_jupyter/__main__.py

```diff
@@ -8,8 +14,40 @@ def submit(file: str):
     """
     Creates new JupyterLab pods defined in the submission file
     """
+    submit_worker(file, ArmadaClient)
+
+
+def submit_worker(file: str, armada_client: ArmadaClient):
+    """
+    A worker function for submit command.
+
+    This function is separated from submit command to allow for easier testing.
+    """
+
+    typer.echo(f"Getting Submission Objects from {file}")
+    submission = submissions.convert_to_submission(file)
+
+    typer.echo(f"Submitting {len(submission.jobs)} Jobs to Armada")
+
+    for job in submission.jobs:
+
+        client, job_id = armada.submit(submission, job, armada_client)
+        typer.echo(f"Submitted Job {job_id} to Armada")
+
+        # Sleep to make sure that job-set-id is created
+        time.sleep(3)
+
+        err = check_job_status(client, submission, job_id)
+
+        if not err:
+            typer.echo(f"Job {job_id} failed to start")
```

Comment:
I would probably suggest logging the actual error if you have it.

Reply:
This is a badly named variable, I will change it as its just a bool